common/src/main/java/com/revenuecat/purchases/common/caching/DeviceCache.kt

```diff
@@ -35,6 +35,7 @@ open class DeviceCache(
     companion object {
         private const val CUSTOMER_INFO_SCHEMA_VERSION_KEY = "schema_version"
         private const val CUSTOMER_INFO_VERIFICATION_RESULT_KEY = "verification_result"
+        private const val CUSTOMER_INFO_REQUEST_DATE_KEY = "customer_info_request_date"
```

Comment:
I gave it this name since there is already a `request_date` which is part of the body of the response. When caching it, I didn't want to override that value, so I serialize it as this key.